sci-physics/geant-vmc/geant-vmc-4.3.6.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils versionator
+
+if [[ ${PV} == *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI=" http://root.cern.ch/git/geant4_vmc.git"
+else
+	DOWN_PV=$(get_version_component_range 2-)
+	SRC_URI="http://root.cern.ch/download/vmc/geant4_vmc.${DOWN_PV}.tar.gz"
+	SOURCE_PV=$(replace_all_version_separators - ${DOWN_PV})
+	S="${WORKDIR}/geant4_vmc-${SOURCE_PV}"
+	KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+fi
+
+DESCRIPTION="Virtual Monte Carlo Geant4 implementation"
+HOMEPAGE="http://root.cern.ch/root/vmc/VirtualMC.html"
+
+LICENSE="GPL-2"
+SLOT="4"
+IUSE="doc examples geant3 +g4root +mtroot vgm test"
+
+RDEPEND="
+	sci-physics/root:=
+	>=sci-physics/geant-4.10.03:=[opengl,geant3?]
+	vgm? ( >=sci-physics/vgm-4.4:= )"
+DEPEND="${RDEPEND}
+	doc? ( app-doc/doxygen )"
+
+DOCS=(
+	history
+	README.md
+)
+
+src_configure() {
+	local mycmakeargs=(
+			-DGeant4VMC_USE_VGM="$(usex vgm)"
+			-DGeant4VMC_USE_GEANT4_G3TOG4="$(usex geant3)"
+			-DGeant4VMC_USE_G4Root="$(usex g4root)"
+			-DGeant4VMC_BUILD_MTRoot="$(usex mtroot)"
+			-DGeant4VMC_BUILD_EXAMPLES="$(usex test)"
+			-DGeant4VMC_INSTALL_EXAMPLES="$(usex examples)"
+			)
+	cmake-utils_src_configure
+}
+
+src_compile() {
+	cmake-utils_src_compile
+	if use doc ; then
+		local dirs=(
+				source
+				$(usev g4root)
+				$(usev mtroot)
+				$(usev examples)
+			)
+		local d
+		for d in "${dirs[@]}"; do
+			pushd ${d} > /dev/null || die
```

Comment:
Please quote path variables whenever possible, especially that you quoted it above.